ai/service/vector_service.py

```
@@ -0,0 +1,138 @@
+            #chroma db는 기본적으로 코사인 유사도 방식
+            similar_docs = self.repository.similarity_search(
+                query=choice,
+                book_id=book_id,
+                k=self.config.search_k
+            )
+            return similar_docs
+        except Exception as e:
+            print(f"유사도 검색 중 오류 발생: {str(e)}")
+            return []
+
+    #벡터 저장    
```

Flake8 (7.2.0)

[error] block comment should start with '# '

(E265)